src/components/home-page/HomeHeroSection.tsx

```diff
@@ -5,16 +5,14 @@ const HomeHeroSection = () => {
   return (
     <div className="flex flex-col items-center gap-10 my-32">
       <section className="text-balance">
-        <h1 className="font-bold text-center text-2xl sm:text-4xl md:text-6xl">
-          Streamline testing with Wopee.io
+        <h1 className="font-bold text-center text-5xl sm:text-4xl md:text-6xl">
```

Comment:
Tailwind is mobile first, so now you made all the small screens have ```text-5xl``` and on sm it becomes ```text-4xl``` and on md and up it becomes ```text-6xl```